Dockerfile

```diff
@@ -1,49 +1,57 @@
-FROM postgres:14.2-alpine AS base
+FROM alpine:3.14
 
 ENV USER hocs
 ENV USER_ID 1000
 ENV GROUP hocs
-ENV NAME hocs-toolbox
-ENV PGDATA /app/data
+ENV NAME hocs-migration-toolbox
 ENV AWS_CLI_VERSION 1.16.207
 
 WORKDIR /app
 
+RUN apk update && apk add bash
+
 RUN addgroup ${GROUP} && \
-    adduser -u ${USER_ID} -G ${GROUP} -h /app -D ${USER}
-RUN mkdir -p /app/scripts && \
-    chown -R ${USER}:${GROUP} /app
-RUN mkdir -p /app/scripts
+    adduser -u ${USER_ID} -G ${GROUP} -h /app -D ${USER} &&\
+    mkdir -p /app/scripts && \
+    chown -R ${USER}:${GROUP} /app &&\
+    mkdir -p /app/scripts
 
 COPY run.sh /app/
 RUN chmod a+x /app/run.sh
 
-
-# Get latest versions of everything
-RUN apk update && apk upgrade
-
-# Installation of AWS CLI & libpq (latter needed for password auth on PG client)
 RUN apk --no-cache update && \
-    apk add --update --no-cache curl py-pip libpq && \
+    apk add --update --no-cache curl py-pip && \
     apk --no-cache add py-setuptools ca-certificates groff less && \
+    apk --no-cache add curl gnupg && \
```

Comment:
Adding curl twice here?

Reply:
Might be worth combining this all into one big `apk add` call actually, instead of running three different ones

Reply:
I've combined the add into one line.